README.md

```diff
@@ -1,23 +1,31 @@
 
-# Falco Contrib
+# Falco Project Evolution
 
-This is a place for various community-maintained resources. In particular, this repository provides a space for the community to work, document, and build out third-party integrations in a safe, and productive way. 
+This repo aims to document the evolution process of The Falco Project.
 
-The intention is to give users easier access to them but to keep them separated from the small scope of the officially supported [Falco's artifacts](https://github.com/falcosecurity/falco).
+## Levels
 
-## Repo structure
+### Sandbox (/contrib)
 
-- [`/deploy`](deploy/) - deployment resources
-- [`/examples`](examples/) - various examples
-- [`/integrations`](integrations/) - third-party integrations
-- *More will come soon!*
+The "*Sandbox*" level serves as a place for the community to test-drive ideas/projects/code.
+Resources with this status can be found under [/contrib](contrib/).
 
-> When adding a new top-level directory, propose your motivations to the [maintainers](OWNERS).
+### Incubating (/repo)
 
-## Join the Community
+The "*Incubating*" level refers to those projects (usually promoted from `/contrib`) that need their repository.
+
+This status is assigned as needed, and can best be measured by the need to cut a release and use the GitHub release features.
+
+**TODO**: `/repo` directory that defines our repositories and `OWNERS`.
```

Comment:
An incubating project will have its own repository by definition. So why having a folder here?!

Reply:
In my opinion, as @nibalizer said in [issue #14](https://github.com/falcosecurity/contrib/issues/14), I'd keep it centralized with other "centralized" data, as for [prow configs](https://github.com/falcosecurity/test-infra/blob/master/prow/config/configs.yaml). I don't know if that's the point.

Reply:
As per #14 :
> A `/repo` directory (maybe with YAML like Kubernetes uses for slack channels) that defines our repositories and OWNERS

That being said, I agree with you that it can be centralized and automated elsewhere so I'm ok with not having `/repo` here.

Reply:
Since we already have an auditable list of projects in `test-infra/prow` I dont'see much value in duplicating it also here and having to maintain both.

Just that...

Reply:
I agree, I will fix it.